### PR TITLE
Problem (Fix #192): keypackage expiration handling not described in detail

### LINKED ISSUE #192: Problem: keypackage expiration handling not described in detail (genesis param, removal...)

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- add the details

### CODE REVIEW

File: docs/modules/tdbe.md [hunk 1]
```diff
@@ -449,10 +449,21 @@ the next node's TDBE (next non-empty leaf) should generate and broadcast `Commit
 and include Remove proposals as with the "commit timeout" case.
 
 ###### Updates
-After 1/3 of keypackage's lifetime is over, TDBE is allowed to generate and broadcast `SelfUpdateProposal`
+When `keypackage.create_time + keypackage_expiration_secs > block_time >= keypackage.create_time + keypackage_update_secs`, TDBE is allowed to generate and broadcast `SelfUpdateProposal`.
 
--- it may happen that there's another committed `SelfUpdateProposal` (from a different sender) or `CommitRemoveTx`,
-which makes TDBE's original `SelfUpdateProposal` -- in which case, it should re-generate and retry.
+When there are concurrently pending mls commits, the operation will fail, so the operator should monitor the operation result, and retry when it failed.
```
Comment (tomtau) [inline docs/modules/tdbe.md]: retries can be handled automatically by TDBE -- operator would only need to sign with the staking key on re-join

**Review by tomtau — CHANGES_REQUESTED** [review]
looks ok -- it'd be good to add the new "keypakcage_update_secs" parameter to https://github.com/crypto-com/chain-docs/pull/204/files#diff-d4f1ae2b8a855e88f80fea911f20aea1 too

File: docs/modules/tdbe.md (hunk 1 above)
Comment (yihuang) [reply docs/modules/tdbe.md:454]: should the TDBE always wait for confirm(buried in block) of previous commit before sending next commit?

Comment (yihuang) [reply docs/modules/tdbe.md]: done

**Comment by yihuang** [comment]
> looks ok -- it'd be good to add the new "keypakcage_update_secs" parameter to https://github.com/crypto-com/chain-docs/pull/204/files#diff-d4f1ae2b8a855e88f80fea911f20aea1 too

done

**Review by tomtau — APPROVED** [review]

**Pull request merged**